app/models/identity.rb

```diff
@@ -1,14 +1,31 @@
 class Identity < ApplicationRecord
   belongs_to :user
   has_many  :backup_data, as: :instance, class_name: "BackupData", dependent: :destroy
+
   validates :uid, :provider, presence: true
   validates :uid, uniqueness: { scope: :provider }, if: proc { |identity| identity.uid_changed? || identity.provider_changed? }
   validates :user_id, uniqueness: { scope: :provider }, if: proc { |identity| identity.user_id_changed? || identity.provider_changed? }
+  # TODO: [thepracticaldev/oss] put the providers somewhere else
   validates :provider, inclusion: { in: %w[github twitter] }
 
+  # TODO: [thepracticaldev/oss] should this be transitioned to JSON?
```

Comment:
I wouldn't mind a `jsonb` column over YAML, but we'd have to migrate all the old ones.

Reply:
me too, i'll leave the todo there for a while. we can also add a new column more aptly named and then transition the data, without having to change type to this one